src/components/material-grid-related/MaterialGridRelatedSelect.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import iconTriangle from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-triangle.svg";
+import {
+  MaterialGridFilterOption,
+  MaterialGridFilterType
+} from "./MaterialGridRelated.types";
+import { useText } from "../../core/utils/text";
+
+type MaterialGridRelatedSelectProps = {
+  filter: MaterialGridFilterType;
+  onChange: (value: MaterialGridFilterType) => void;
+  options: MaterialGridFilterOption[];
+};
+
+export function MaterialGridRelatedSelect({
+  filter,
+  onChange,
+  options
+}: MaterialGridRelatedSelectProps) {
+  const t = useText();
+
+  return (
+    <div className="dropdown dropdown--grey-borders input-with-dropdown__dropdown material-grid-related__dropdown">
```

Comment:
Please consider whether we can use the existing [`Dropdown`](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/3b69c39a82e283ff5722168ccb6761cf559a5ab9/src/components/Dropdown/Dropdown.tsx) component here.